lab-ben/test/server-test.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const expect = require('chai').expect;
+const chat = require('./../server.js');
+const net = require('net');
+
+describe('Chat connection', function() {
+  beforeEach(done => {
+    chat.listen(3000);
+    done();
+  });
+  afterEach(done => {
+    chat.close(3000);
+    done();
+  });
+
+  describe('#join chat', function() {
```

Comment:
FYI, semantically, using ``#`` denotes you're testing an actual function itself, and you're not technically doing that here, so no need to put an extraneous ``#`` since your description of ``join chat`` is explicit enough.